index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel= "https://fonts.googleapis.com/css2?family=Inter&display=swap" href="style.css">
+  <title> My portfolio</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <nav class="container">
+      <ul>
+        <li><a><h2 class="upitem">Welcome </h2></a></li>
+        <li><img src="./imges/Union.svg" alt="menu"></li>
+      </ul>
+    </nav>
+  </header>
+  <section class="tittleContents"> 
+    <h1 class="welcome"> Hey there. I&apos;m Gabriel Cilfone<br><span class = "desciption">I&apos;m a software developer</span></h1>
```

Comment:
- [ ] I would suggest you instead of putting this text that says `I'm a software developer` in the `h1`, you take it out in an `h2` tag.  And please make sure that it has the same colour, font and font size compared to the design template in Figma as well as your `h1` and your paragraph. 👌
- [ ] Please make sure that your design is similar compared to the Figma design template because right now they are different.

### Your design 
![Screenshot from 2022-10-31 23-35-59](https://user-images.githubusercontent.com/89228743/199129173-a534710d-e5de-4baf-acf2-63445a835e21.png)

### The Figma design

![Screenshot from 2022-10-31 23-36-33](https://user-images.githubusercontent.com/89228743/199129219-f157ca68-0ea5-4bf1-875d-dc8d49f773b5.png)

Reply:
Fixed.